maquetado/scss/application.scss

```diff
@@ -0,0 +1,3 @@
+.title {
+  color: red;
```

Comment:
Later we store things like color in variables inside a `color.scss` file and make the import statement in this file